flathub.json

```diff
@@ -0,0 +1,3 @@
+{
+	"only-arches": ["x86_64"],
```

Comment:
This is not the fix.

Either you take a more recent hdf5 that build out of the box, or you apply the usual "outdated configure" fix: adding a `shell` source that will do `cp -p /usr/share/automake-*/config.{sub,guess} .`.

Reply:
As far as I know there is no working aarch64 build of kst. I struggled a lot with adding support for the hdf5. Recent versions will not build for me on x64 either. Is support for aarch64 mandatory for publishing? If so, this pull request shall be closed and the work will be postponed for the forseable future as I have limited time to put in to this specific project.

Reply:
how about 
> or you apply the usual "outdated configure" fix: adding a shell source that will do `cp -p /usr/share/automake-*/config.{sub,guess} .`.

Reply:
there are many reason why I'd be ok for skipping aarch64, but this one ain't one.

Reply:
This specific error or the fact that no one has compiled it for aarch64 before? 

This is my first time contributing a flatpak so "the usual outdated configure" does not tell me that much, could you give me some more pointers? Adding a shell source where?

Reply:
as for "never built", if you don't try then it will never happen.

Reply:
That it true, fingers crossed for this build :)